### PR TITLE
Remove redundant code

### PR DESCRIPTION
**Pull request by ebyhr**
# Description

Remove redundant code

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

Please delete options that are not relevant.

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] If adding new functionality, I have discussed my implementation with the community using the linked GitHub issue
- [x] I have signed and submitted the [ICLA](https://github.com/polaris-catalog/polaris/blob/main/ICLA.md) and if needed, the [CCLA](https://github.com/polaris-catalog/polaris/blob/main/CCLA.md). See [Contributing](https://github.com/polaris-catalog/polaris/blob/main/CONTRIBUTING.md) for details.

### CODE REVIEW

File: polaris-service/src/main/java/io/polaris/service/auth/OAuthUtils.java [hunk 1]
```diff
@@ -16,31 +16,11 @@
...
 import jakarta.ws.rs.core.Response;
-import java.nio.charset.StandardCharsets;
-import org.apache.commons.codec.binary.Base64;
 
 /** Simple utility class to assist with OAuth operations */
 public class OAuthUtils {
-
-  public static final String AUTHORIZATION_HEADER = "Authorization";
-
-  public static final String SF_HEADER_ACCOUNT_NAME = "Snowflake-Account";
```
Comment (takidau) [inline polaris-service/src/main/java/io/polaris/service/auth/OAuthUtils.java:27]: This looks like a Snowflake specific string that arguably doesn't need to live here any more, but first I'd like to verify nothing we have internally depends on it before removing it. Let me check on that.

Comment (takidau) [reply polaris-service/src/main/java/io/polaris/service/auth/OAuthUtils.java:27]: Confirmed, we still use these internally. Let me try to quickly remove our internal dependency on these and then we can move forward here.

**Comment by ebyhr** [comment]
(Rebased on main to resolve conflicts)

File: polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java [hunk 2]
```diff
@@ -1732,8 +1721,7 @@ private void updateTableLike(long catalogId, TableIdentifier identifier, Polaris
     }
   }
 
-  private @NotNull PolarisMetaStoreManager.DropEntityResult dropTableLike(
-      long catalogId,
+  private PolarisMetaStoreManager.@NotNull DropEntityResult dropTableLike(
```
Comment (snazy) [inline polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java:1724]: What's this change for?

File: polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java [hunk 3]
```diff
@@ -1664,14 +1656,11 @@ private void createTableLike(long catalogId, TableIdentifier identifier, Polaris
           String.format("Failed to fetch resolved parent for TableIdentifier '%s'", identifier));
     }
 
-    createTableLike(catalogId, identifier, entity, resolvedParent);
+    createTableLike(identifier, entity, resolvedParent);
```
Comment (snazy) [inline polaris-service/src/main/java/io/polaris/service/catalog/BasePolarisCatalog.java:1659]: `catalogId` is unused in this function

File: polaris-service/src/main/java/io/polaris/service/auth/OAuthUtils.java (hunk 1 above)
Comment (takidau) [reply polaris-service/src/main/java/io/polaris/service/auth/OAuthUtils.java:27]: Sorry for the delay. This is all cleaned up on our side now.

**Review by takidau — APPROVED** [review]
LGTM, thank you for your patience!

**Comment by takidau** [comment]
Apparently someone else needs to review now since I resolved against other changes... :-/ @snazy if you get a chance maybe?

**Review by snazy — APPROVED** [review]
Sure

**Pull request merged**